dashboard/src/components/Catalog/CatalogItem.tsx

```diff
@@ -4,6 +4,7 @@ import placeholder from "../../placeholder.png";
 import { IChart } from "../../shared/types";
 import InfoCard from "../InfoCard";
 
+import { Link } from "react-router-dom";
```

Comment:
really minor nitpick: can we move this import to group it with the react import? I like having the library imports grouped separately from the local imports.